examples/BUILD-EXAMPLES.md

```diff
@@ -14,25 +15,19 @@
 
 1. Navigate to the examples directory and use `cmake` to build all examples. For non-Windows systems, the commands are:
 
-    ```
-    $ cd workspace/examples
-    $ mkdir build
-    $ cd build
-    $ cmake .. -DGEODE_ROOT="<geode-install-root-dir>"
-    $ cmake --build . -- <optional parallelism parameter>
-    ```
-  where the optional parallelism parameter is of the form `-j n`.
+       $ cd workspace/examples
```

Comment:
What is the rational for this formatting change. The code block formatting is more appropriate than the fixed width indention.

Reply:
Less is more. The rendered output looks exactly the same. Also, it's not really code, and it makes the markdown clearer because we know the indentation is part of the list item.

Reply:
Less is more? How are 6 backticks suddenly more than N x 4 spaces, where N is the number of lines > 1? 

Formats the same? Not sure where it go dropped but this code blocks had a language of console for proper highlighting. 
``````
```console
$ command --args
```
``````
produces
```console
$ command --args
```

Reply:
The documentation was all over the place and almost none of it had any console tag associated with it. The *one* I did see that on didn't format the text any different than a gray box just like all the rest. If it had formatted like you imply, I would have kept with the backticks. If it turns out Github at least does properly format the console tag, I'll add it.

But backticks are otherwise mysterious, indentation is intuitive. Markdown has indent enabled formatting for this very purpose - it's no coincidence that the right thing occurs when you properly indent in markdown.

Reply:
Although indentation is somewhat easier to read in the markdown, we're usually looking at the rendered output. So readability may be less important than being consistent in our docs. We seem to mostly use back tics. Also, I like the color coding afforded by using using console.

WINNER: Back tics, with console attribute.